### PR TITLE
G's initial mods

### PR DESCRIPTION
**Pull request by oryktos**
Initial-state modifications:

* Reorder non-bot function definitions

* Separate sections using horizontal

* Add main function

* Add entrypoint

* Change algorithm of `make_upload_packages`

### CODE REVIEW

File: meme_machine.py [hunk 1]
```diff
@@ -1,72 +1,126 @@
...
+
+# 8MB max
+MAX_MEME_SIZE = 8 * 1024 * 1024
+
+
+# Function definitions
+# -----------------------------------------------------------------------------
+
+
+def get_size(file_path: str):
```
Comment (TheSpectacledOne) [inline meme_machine.py]: Remove this...

File: meme_machine.py [hunk 2]
```diff
@@ -1,72 +1,126 @@
...
+    return output
+
+
+def read_token(file_path: str) -> str:
+    with open(file_path, "r") as fh:
+        data = fh.read()
+    return data.strip()
+
+
+def under_8MB(cur_size, file_size):
```
Comment (TheSpectacledOne) [inline meme_machine.py]: ...and this.

File: meme_machine.py [hunk 3]
```diff
@@ -1,72 +1,126 @@
...
+    files_in_dir = []
+    output = [[]]
+    for filename in os.listdir(dir_path):
+        full_path = os.path.join(dir_path, filename)
+        # Skip directories
+        if os.path.isfile(full_path):
+            # Skip large files
+            size = os.path.getsize(full_path)
+            if size <= MAX_MEME_SIZE:
+                files_in_dir.append((full_path, size))
```
Comment (TheSpectacledOne) [inline meme_machine.py]: Add the discord.File objects here!

File: meme_machine.py (hunk 1 above)
Comment (oryktos) [reply meme_machine.py]: ✔

**Review by TheSpectacledOne — CHANGES_REQUESTED** [review]
I like most of the changes!

File: meme_machine.py (hunk 2 above)
Comment (oryktos) [reply meme_machine.py]: ✓

File: meme_machine.py (hunk 3 above)
Comment (oryktos) [reply meme_machine.py]: ✔

**Pull request merged**